unattended_installer/common_functions/common.sh

```diff
@@ -56,7 +56,7 @@ function common_logger() {
     fi
 
     if [ -z "${debugLogger}" ] || { [ -n "${debugLogger}" ] && [ -n "${debugEnabled}" ]; }; then
-        if [ "$EUID" -eq 0 ] && [ -z "${nolog}" ]; then
+        if { [ "$EUID" -eq 0 ] && [ -z "${nolog}" ]; } || [[ "$(basename "$0")" =~ $cert_tool_script_name ]]; then
```

Comment:
Notice that, when the script is the cert tool, the `nolog` variable is not taken into account.
So, it means that, in the cert tool, it will write every message in the logfile even if the message is marked as "no log"